README.md

```diff
@@ -262,6 +262,8 @@ The devices below are fully feature supported by OpenRazer, which means all avai
 | Razer Tartarus V2                             |  1532:022B  |
 | Razer Nommo Chroma                            |  1532:0517  |
 | Razer Nommo Pro                               |  1532:0518  |
+| Razer Mouse Dock                              |  1532:007E  |
+| Razer Mouse Dock Pro                          |  1532:00A4  |
```

Comment:
Keep the list sorted by PID at least, but probably both Mouse Dock and the new Mouse Dock Pro should go into the accessories table?

Reply:
There's no dedicated accessories table. All accessories currently live in this "Misc" table.

Should an Accessories Table be created?

Reply:
Misc is totally fine, there's too many random Razer products to categorize them perfectly, but we also have nice categorization on the website so it's fine to not be too granular here.